prow/plugins/project/project.go

```diff
@@ -114,22 +114,42 @@ func updateProjectNameToIDMap(projects []github.Project) {
 // proposed column name, whether the command is to remove issue/PR from project,
 // and the error message
 func processRegexMatches(matches []string) (string, string, bool, string) {
```

Comment:
We can send in just `matches[1]` since the project and column names will be grouped together now

Reply:
done